### PR TITLE
(PCP-792) fix test on AIX and solaris

### PR DESCRIPTION
**Pull request by zreichert**

### CODE REVIEW

File: acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb [hunk 1]
```diff
@@ -8,22 +8,29 @@
     skip_test "Test is not compatible with #{platform}" if platform =~ /windows/
 
     step 'create non-root user on all nodes' do
-      @user_home_dir = '/foo'
+      @user_name = 'foo'
+      @group_name = 'foobar'
+      (platform =~ /solaris/) ? @user_home_dir = "/export/home#{@user_name}" : @user_home_dir = "/home/#{@user_name}"
```
Comment (MikaelSmith) [inline acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb]: How about `@user_home_dir = platform =~ /solaris/ ? "/export/home#{@user_name}" : "/home/#{@user_name}"`?

File: acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb [hunk 2]
```diff
@@ -8,22 +8,29 @@
...
       @user_puppetlabs_dir = "#{@user_home_dir}/.puppetlabs"
       @user_puppet_dir = "#{@user_puppetlabs_dir}/etc/puppet"
       @user_pxp_dir = "#{@user_puppetlabs_dir}/etc/pxp-agent"
       on(agent, "mkdir -p #{@user_puppet_dir}")
       on(agent, "mkdir -p #{@user_pxp_dir}")
-      on(agent, puppet("resource group foo ensure=present"))
-      on(agent, puppet("resource user foo ensure=present home=#{@user_home_dir} groups=foo"))
+      on(agent, puppet("resource group #{@group_name} ensure=present"))
+      if platform =~ /eos/
+       #arista
```
Comment (MikaelSmith) [inline acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb]: Indentation's funny here.

File: acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb (hunk 1 above)
Comment (zreichert) [reply acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb]: Thats much cleaner, I guess that's what I get for letting my IDE refactor my code.  :)

File: acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb (hunk 2 above)
Comment (zreichert) [reply acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb]: fiex

**Pull request merged**